scripts/import/import_clinvar_xml

```diff
@@ -108,8 +110,21 @@ clean_old_data() if defined $clean;
 my $omim_set_id   = get_set($dbh, 'ph_omim');
 my %omimVarSet;
 
+my $pheno_set_id   = get_set($dbh, "ph_variants"); #All phenotype/disease variants
+my $clinvar_set_id = get_set($dbh, "ClinVar"); #All ClinVar variants
+
 ## default ClinVar phenotype description if non available
 my $default_pheno  = "ClinVar: phenotype not specified";
+my $non_specified_class = 663;
```

Comment:
Ideally, we'd look this up from a text string rather than have the id hard-coded